src/index.html

```diff
@@ -3,12 +3,239 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Title</title>
+    <title>Uber Eats</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="./styles/main.css">
   </head>
   <body>
-    <img src="./images/logo.svg" alt="Logo">
-    <h1>Hello Mate Academy</h1>
-    <script type="text/javascript" src="scripts/main.js"></script>
+
+    <header class="header" id="to-top">
+      <a href="#">
+        <img class="header__logo" src="./images/logo.svg" alt="logo">
+      </a>
+
+      <div class="header__left-side">
+        <a href="#" class="header__left-side header__address-time">asap</a>
+        <span class="text">to</span>
+        <form class="header__left-side-place-form">
+          <input class="header__left-side-place" type="text" value="ul. Tarasivska St, 16" placeholder="Enter your address here" required>
```

Comment:
Делаем переносы для длинных тегов
```suggestion
          <input
            class="header__left-side-place"
            type="text"
            value="ul. Tarasivska St, 16"
            placeholder="Enter your address here"
            required
          >
```